scripts/sql/161_generic_note.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER SEQUENCE IF EXISTS id_seq_cluster_note RENAME to id_seq_generic_note;
+ALTER SEQUENCE IF EXISTS id_seq_cluster_note_history RENAME to id_seq_generic_note_history;
+AlTER TABLE cluster_note DROP CONSTRAINT cluster_note_cluster_id_fkey;
+ALTER TABLE cluster_note RENAME to generic_note;
+ALTER TABLE generic_note RENAME COLUMN cluster_id to identifier;
+ALTER TABLE generic_note ADD COLUMN  identifier_type int4 DEFAULT 0 NOT NULL;
```

Comment:
we don't set Default values to existing tables, we should handle it through code.